File: src/test/java/nl/mvdr/adventofcode/adventofcode2024/day03/Part2Test.java
package nl.mvdr.adventofcode.adventofcode2024.day03;

import java.util.stream.Stream;

import org.junit.jupiter.params.provider.Arguments;

import nl.mvdr.adventofcode.SolverTest;

public class Part2Test extends SolverTest<Part2> {

    public Part2Test() {
        super(Part2.class);
    }
    
    static Stream<Arguments> testSolution() {
        return Stream.of(
                Arguments.of("48", "example-day03-2024-1.txt"));
    }
}

File: adventofcode-2025/src/test/kotlin/nl/mvdr/adventofcode/adventofcode2025/day03/Part2Test.kt
package nl.mvdr.adventofcode.adventofcode2025.day03

import nl.mvdr.adventofcode.solver.FunctionSolverTest
import org.junit.jupiter.params.provider.Arguments

class Part2Test: FunctionSolverTest<Long>(::solvePart2) {
    companion object {
        @JvmStatic
        fun testSolution(): List<Arguments> {
            return listOf(
                Arguments.of("3121910778619", "example-day03.txt")
            )
        }
    }
}